distutils/_msvccompiler.py

```diff
@@ -533,3 +556,10 @@ def find_library_file(self, dirs, lib, debug=0):
         else:
             # Oops, didn't find it in *any* of 'dirs'
             return None
+
+
+class ClangMSVCCompiler(MSVCCompiler):
+    compiler_type = 'clang-cl'
```

Comment:
The name `clang-cl` seems redundant, especially if `cl` is short for `clang`. Where is this name defined?

Reply:
`cl` is not short for `clang`. `cl` is for the MSVC compiler. There's a `clang-cl.exe` produced by the clang team that mimicks the behaviour of the MSVC compiler `cl`.